.gitignore

```diff
@@ -22,3 +22,4 @@
 .yardoc/
 /doc/
 /failures
+.DS_Store
```

Comment:
Good point, and we're bike shedding at this point anyway. It does no harm to have this file ignored, and almost all of our developers are on Macs. Happy for this to remain 👍